examples/pytorch/rgcn/entity_sample_cugraph_relgraphconv.py

```diff
@@ -0,0 +1,179 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torchmetrics.functional import accuracy
```

Comment:
Do we have to use torchmetrics?
Is torchmetrics a dependency of DGL?
@mufeili

Reply:
It's not a dependency. However @jermainewang previously suggested using it so we can avoid writing some lines of code for metric computation ourselves.

Reply:
Yeah, it saves many codes for computing standard metrics like accuracy.